sdk/core/Microsoft.Azure.Core.Spatial.NewtonsoftJson/src/NewtonsoftJsonGeographyConverter.cs

```diff
@@ -8,12 +8,13 @@
 namespace Azure.Core.Serialization
 {
     /// <summary>
-    /// Converts between <see cref="GeographyPoint" /> objects and Geo-JSON points.
+    /// Converts between <see cref="Geography" /> objects and Geo-JSON points.
     /// </summary>
-    public class NewtonsoftJsonGeographyPointConverter : JsonConverter
+    public class NewtonsoftJsonGeographyConverter : JsonConverter
```

Comment:
Add tests to make sure that both converters show actionable messages including line, column, offset, path, etc: https://github.com/Azure/azure-sdk-for-net/pull/15387#discussion_r496977631

Reply:
Turns out Newtonsoft.Json only contains more detail in 12.0.1 or newer. I will at least look at improving the message similar to what I did for STJ, but if someone is using a newer Newtonsoft.Json they will get this for free. I don't think it's worth duplicating the behavior ourselves.

Reply:
Agreed.  I think Upgrade Newtonsoft is an acceptable answer for better debugging given the specific scenarios here.

Reply:
Talking offline, `System.Text.Json` will have this info, but not `Newtonsoft.Json`. Converters have to supply line information themselves starting with version 12.0.1, which we can't do without reflection and aren't there yet. I opened Newtonsoft.Json#2395 to see if newer versions could at least set this information automatically if not already set. Offline, we decided not to do anything for this case for now.